public_html/style.css

```diff
@@ -199,6 +222,489 @@ select.error, textarea.error, input.error {
+    font-size: 12px;
+    font-weight: bold;
+    line-height: 20px;
+    color: #00A0E2;
+    color: rgb(0, 160, 226);
+}
+
+.highlightedInfo {
+    padding-bottom: 12px;
+}
+
+.infoRowTitle {
+    display: inline-block;
+    width: 50%;
+}
+
+.infoRowContent {
+    display: inline-block;
+    width: 40%;
+}
+
+.infoRowFluid {
+    display: inline-block;
+}
+
+.infoRow {
+    padding-top: 10px;
+}
+
+.removePadding {
+    padding: 0;
+}
+
+.rightLink {
+    float: right;
+    padding-right: 5px;
+}
+
+.infoBlockTopSection {
+    padding-top: 14px;
+    padding-bottom: 14px;
+    padding-left: 8px;
+}
+
+.infoBlock45pxSection {
+    height: 45px;
+    line-height: 45px;
+    padding-left: 10px;
+}
+
+.infoBlockSection {
+    padding-top: 14px;
+    padding-bottom: 14px;
+    padding-left: 10px;
+}
+
+.lightGreyBackground {
+    background-color: #efefef;
+}
+
+.fourColumnSection1 {
+    width: 20%;
+}
+
+.fourColumnSection2 {
+    width: 20%;
+}
+
+.fourColumnSection3 {
+    width: 20%;
+}
+
+.fourColumnSection4 {
+    width: 20%;
+}
+
+#dump1090_infoblock {
+    font-family: Helvetica Neue;
+}
+
+.infoBlockTitleText {
+    font-weight: bold;
+    color: #002F5D;
+}
+
+.legendBox {
+    width: 15px;
+    height: 15px;
+    border: 1px solid #efefef;
+    border-radius: 4px;
+    -moz-border-radius: 4px;
+    -webkit-border-radius: 4px;
+}
+
+#adsbLegendBox {
+    background-color: #E5F6FC;
+}
+
+#mlatLegendBox {
+    background-color: #FDF2E5;
+}
+
+#otherLegendBox {
+    background-color: #E6E6E6;
+}
+
+#tisbLegendBox {
+    background-color: #CCD5DF;
+}
+
+.legendTitle {
+    /*margin-left: auto;*/
+    line-height: 19px;
+    display: inline-block;
+    padding-right: 20px;
+    padding-left: 5px;
+}
+
+#settings_infoblock {
+    position: absolute;
+    right: 1px;
+    top: 60px;
+    width: 398px;
+    min-height: 180px;
+    background: #ffffff;
+    box-shadow: 4px 4px 10px #444444;
+    padding: 20px;
+    z-index: 9999;
+    display: none;
+}
+
+.settingsColumn {
+    display: table-cell;
+    width: 199px;
+}
+.settingsOptionContainer {
+    display: table;
+    padding-bottom: 10px;
+}
+
+.settingsCheckbox {
+    width: 20px;
+    height: 11px;
+    background-image: url('images/box-empty.png');
+    background-repeat: no-repeat;
+    background-position: center;
+    cursor: pointer;
+}
+
+.settingsCheckboxChecked {
+    background-image: url('images/box-checked.png') !important;
+}
+
+.settingsCloseBox {
+    position: absolute;
+    right: 8px;
+    top: 8px;
+    background-image: url('images/close-settings.png');
+    width: 20px;
+    height: 20px;
+    cursor: pointer;
+}
+
+.settingsText {
+    line-height: 20px;
+    display: table-cell;
+}
+
+#altitude_chart {
+    float: right;
+    width: calc(100% - 5px);
+    max-width: 800px;
+    right: 5px;
+    bottom: 35px;
+    background-color: transparent !important;
+}
+
+#altitude_chart_button {
+    background-image: url('images/alt_legend_feet.svg');
```

Comment:
If the user changes the color configuration, the aircraft will change to reflect their modifications but the legends won't.